cli/src/installers/tailwind.ts

```diff
@@ -26,7 +26,10 @@ export const tailwindInstaller: Installer = ({ projectDir }) => {
   const postcssCfgSrc = path.join(extrasDir, "config/postcss.config.cjs");
   const postcssCfgDest = path.join(projectDir, "postcss.config.cjs");
 
-  const prettierSrc = path.join(extrasDir, "config/prettier.config.cjs");
+  const prettierSrc = path.join(
+    extrasDir,
+    "config/prettier-with-tailwind.config.cjs",
+  );
```

Comment:
we can then remove prettier from here - keep all the prettier logic in the prettier-eslint installer.